pkg/kv/kvserver/scheduler.go

```diff
@@ -344,13 +348,19 @@ func (s *raftScheduler) worker(ctx context.Context) {
 func (s *raftScheduler) enqueue1Locked(
 	addFlags raftScheduleFlags, id roachpb.RangeID, now int64,
 ) int {
+	// TODO(pavelkalinnikov): Pass ticks increment to avoid branching below.
+	ticked := addFlags&stateRaftTick != 0
```

Comment:
We can make it branchless by dividing by `stateRaftTick`, which will yield 1, which can be added to `newState.ticks`.